mail_activity_team/static/src/scss/systray.scss

```diff
@@ -0,0 +1,9 @@
+.o_mail_systray_item {
```

Comment:
If we remove the `new_notification_counter` class I think we can also remove this file